accurest-core/src/main/groovy/io/codearte/accurest/dsl/internal/Multipart.groovy

```diff
@@ -0,0 +1,41 @@
+package io.codearte.accurest.dsl.internal
+
+import groovy.transform.CompileStatic
+import groovy.transform.EqualsAndHashCode
+import groovy.transform.ToString
+
+@ToString(includePackage = false, includeFields = true, includeNames = true)
+@EqualsAndHashCode(includeFields = true)
+@CompileStatic
+class Multipart extends DslProperty {
+
+    Multipart(Map<String, DslProperty> multipart) {
+        super(extractValue(multipart, { DslProperty p -> p.clientValue}), extractValue(multipart, {DslProperty p -> p.serverValue}))
+    }
+
+    private static Map<String, Object> extractValue(Map<String, DslProperty> multipart, Closure valueProvider) {
+        multipart.collectEntries { Map.Entry<String, DslProperty> entry ->
```

Comment:
yes, it's optional but we prefer to have it explicit (only in small closures we can omit this)